docs/bundles/plain.md

```diff
@@ -2,23 +2,52 @@
 
 ## Overview
 
-This document is meant to define the `plain` bundle format as a reference for those publishing `plain` bundles
-for use with RukPak. For more information on the concept of a bundle, click [here](https://github.com/operator-framework/rukpak#bundle).
+This document is meant to define the plain bundle format as a reference for those publishing plain bundles for use with
+RukPak. A bundle is a collection of Kubernetes resources that are packaged together for the purposes of installing onto
+a Kubernetes cluster. A bundle can be unpacked onto a running cluster, where controllers can then create the underlying
+content embedded in the bundle. The bundle can be used as the underlying `spec.source` for
+a [`Bundle`](https://github.com/operator-framework/rukpak#bundle) resource.
 
-A plain bundle is simply a collection of static, arbitrary, Kubernetes YAML manifests in a given directory.
+A plain bundle is a static collection of arbitrary Kubernetes YAML manifests. A plain bundle is contained in a directory
+that can be packaged in a container image layer , a `git` repository or any other content source that the
+[plain bundle provisioner](../provisioners/plain.md) supports.
+
+Supported source types for a plain bundle currently include the following:
+
+* A directory in a [container image](../sources/image.md)
+* A directory in a [`git` repository](../sources/git.md)
+* A set of keys in a [`ConfigMap`](../sources/local.md)
+* An [upload](../sources/upload.md)
+* A `.tgz` file returned by a [http endpoint](../sources/http.md)
+
+Additional source types (such as a local volume or a generic URI-based resource) are on the roadmap. These source types all present the same content: a directory containing static Kubernetes YAML manifests
```

Comment:
I would drop this line. There's nothing on the immediate roadmap at this point AFAIK.

Reply:
Removed in https://github.com/operator-framework/rukpak/pull/642/commits/56cfbae769c96755690befe3628c45b9e36ee244